pkg/distributors/contracts/MultiRewards.sol

```diff
@@ -90,7 +96,10 @@ contract MultiRewards is IMultiRewards, IDistributor, ReentrancyGuard, Temporari
         address rewarder
     ) external override {
         require(
-            msg.sender == owner() || msg.sender == address(pool) || isAssetManager(pool, msg.sender),
+            msg.sender == owner() ||
+                msg.sender == address(pool) ||
+                msg.sender == address(rewardsScheduler) ||
+                isAssetManager(pool, msg.sender),
```

Comment:
Is this maybe the result of an incomplete refactor from an alternative approach? The scheduler is not adding any rewarders to any whitelist (nor does it need to), I'm not sure why we're letting it do it.